samcli/commands/init/init_templates.py

```diff
@@ -2,23 +2,30 @@
 Manages the set of application templates.
 """
 
+import re
 import itertools
 import json
 import logging
 import os
 from pathlib import Path
 from typing import Dict
 
+
 import click
 
 from samcli.cli.main import global_cfg
 from samcli.commands.exceptions import UserException, AppTemplateUpdateException
 from samcli.lib.utils.git_repo import GitRepo, CloneRepoException, CloneRepoUnstableStateException
 from samcli.lib.utils.packagetype import IMAGE
-from samcli.local.common.runtime_template import RUNTIME_DEP_TEMPLATE_MAPPING, get_local_lambda_images_location
+from samcli.local.common.runtime_template import (
+    RUNTIME_DEP_TEMPLATE_MAPPING,
+    get_local_lambda_images_location,
+)
 
 LOG = logging.getLogger(__name__)
-APP_TEMPLATES_REPO_URL = "https://github.com/aws/aws-sam-cli-app-templates"
+# Note:: This would be changed when after aws-sam-cli-app-templates pr
+# https://github.com/aws/aws-sam-cli-app-templates/pull/127 is merged
```

Comment:
Is there anything preventing merge of https://github.com/aws/aws-sam-cli-app-templates/pull/127

Reply:
@jonife Have you tested that the new manifest also works with the current/develop version of the SAM CLI?

Reply:
Yes I have test and the changes to the manifest file can't affect the how current/develop version of the SAM CLI works as it does not depend on the manifest for it interactive flow but only to spool the application after cloning.